internal/sysfs/sysfs.go

```diff
@@ -68,7 +71,7 @@ type FS interface {
 	// ^^ TODO: Consider syscall.Mkdir, though this implies defining and
 	// coercing flags and perms similar to what is done in os.Mkdir.
 
-	// Rename is similar to syscall.Rename, except the path is relative to this
+	// Rename is similar to os.Rename, except the path is relative to this
```

Comment:
the former was more correct as we are returning syscall.Errno not the wrapped stuff